static/js/subs.js

```diff
@@ -431,6 +431,7 @@ exports.setup_page = function () {
 
     function populate_and_fill(public_streams) {
 
+        exports.all_streams = public_streams.streams;
```

Comment:
Okay. I guess I should add this variable as global in this file and use it later in `populate_and_fill`.